spec/models/post_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] I got this console error when I ran the `rspec `command. Kindly fix it and make sure that all the tests are running without issues.

![Screenshot from 2022-08-12 17-39-05](https://user-images.githubusercontent.com/63087491/184405522-9474e15f-b0ec-453d-92b8-cb988fcddab4.png)


Thanks :pray: